server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/Serializer.scala

```diff
@@ -407,7 +417,10 @@ class Serializer @Inject() (mailboxIdFactory: MailboxId.Factory) {
 
   def serialize(errors: JsError): JsValue = Json.toJson(errors)
 
-  def serialize(vacationResponse: VacationResponse): JsValue = Json.toJson(vacationResponse)
+  def serialize(vacationResponse: VacationResponse)(implicit vacationResponseWrites: Writes[VacationResponse]): JsValue =
```

Comment:
`(implicit vacationResponseWrites: Writes[VacationResponse])` is likely not needed?

Reply:
The result of a bad rebase, thanks :)